src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -235,12 +235,16 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                 ->arrayNode('workflows')
```

Comment:
The issue is the `xsd:all` node which we use as the top-level node of the `config` node. Afaik this doesn't allow us to specifiy several `workflow` nodes. That's why the `workflow` nodes are wrapped inside the `workflows` node and calling `fixXmlConfig()` isn't necessary for the `workflow` config node.